packages/dht/src/dht/discovery/PeerDiscovery.ts

```diff
@@ -139,11 +119,10 @@ export class PeerDiscovery {
         if (this.stopped) {
             return
         }
-        await Promise.allSettled(this.config.bucket.closest(this.config.ownPeerId.value, this.config.parallelism).map(async (peer: RemoteDhtNode) => {
-            const contacts = await peer.getClosestPeers(this.config.ownPeerDescriptor.kademliaId)
-            contacts.forEach((contact) => {
-                this.config.addContact(contact)
-            })
+        await Promise.allSettled(this.config.peerManager.getClosestPeersTo(
+            this.config.ownPeerDescriptor.kademliaId, this.config.parallelism).map(async (peer: RemoteDhtNode) => {
+            const contacts = await peer.getClosestPeers(this.config.ownPeerDescriptor.kademliaId!)
+            this.config.peerManager.handleNewPeers(contacts)
```

Comment:
Just an indentation fix

```suggestion
        await Promise.allSettled(
            this.config.peerManager.getClosestPeersTo(
                this.config.ownPeerDescriptor.kademliaId,
                this.config.parallelism
            ).map(async (peer: RemoteDhtNode) => {
                const contacts = await peer.getClosestPeers(this.config.ownPeerDescriptor.kademliaId!)
                this.config.peerManager.handleNewPeers(contacts)
            })
        )
```